using_channels/subtitles-es.srt

```diff
@@ -0,0 +1,91 @@
+1
+00:00:00,820 --> 00:00:04,780
+La tienda de Snaps brinda cuatro canales en los cuales su software puede ser publicado.
+
+2
+00:00:04,780 --> 00:00:08,040
+Se llaman «stable», «candidate», «beta» y «edge».
+
+3
+00:00:08,460 --> 00:00:11,480
+Cada canal significa un nivel de riesgo para la usuaria.
+
+4
+00:00:11,480 --> 00:00:15,320
+entonces el software publicado ahí debería tener la estabilidad apropiada.
+
+5
+00:00:15,320 --> 00:00:19,980
+Como desarrolladora, usted puede publicar diferentes versiones de su snap en cada canal.
+
+6
+00:00:20,720 --> 00:00:23,580
+Publique de forma automática directamente desde cada «commit» de Github,
+
+7
+00:00:23,740 --> 00:00:27,120
+al canal «edge» desde build.snapcraft.io,
+
+8
+00:00:27,140 --> 00:00:30,620
+desde Travis, o de forma manual usando el comando snapcraft.
+
+9
+00:00:30,620 --> 00:00:32,940
+Tome versiones soportadas de su software
+
+10
+00:00:33,020 --> 00:00:35,640
+y envíelas de forma manual al canal «stable».
+
+11
+00:00:35,640 --> 00:00:38,800
+Usuarias buscando estabilidad sobre las funcionalidades más recientes
+
+12
+00:00:38,960 --> 00:00:40,960
+pueden subscribirse a ese canal.
+
+13
+00:00:40,960 --> 00:00:44,700
+Sus equipos de integración contínua y aseguramiento de la calidad pueden subscribirse a «edge»
+
+14
+00:00:44,700 --> 00:00:47,480
+para obtener las versiones más recientes de forma automática.
+
+15
+00:00:47,660 --> 00:00:49,920
+Durante la preparación de una versión puede usar el canal «beta»
```

Comment:
During prerelease you can use the beta channel > Durante el pre-lanzamiento usted puede usar el <canal beta>